src/jdk.jdwp.agent/unix/native/libjdwp/exec_md.c

```diff
@@ -96,18 +96,19 @@ closeDescriptors(void)
 #endif
 
     if ((dp = opendir(FD_DIR)) == NULL) {
-        LOG_MISC(("warning: failed to open dir %s while determining"
+        ERROR_MESSAGE(("failed to open dir %s while determining"
                   " file descriptors to close for process %d", FD_DIR, getpid()));
```

Comment:
Nit: indent of second line is now off - please align "

Reply:
Done. PR has been update to fix this.